core/src/main/java/org/mapfish/print/servlet/MapPrinterServlet.java

```diff
@@ -322,6 +326,7 @@ public final void createReport(@PathVariable final String appId,
             json.object();
             {
                 json.key(JSON_PRINT_JOB_REF).value(ref);
+                json.key(JSON_PRINT_JOB_POSITION).value(this.jobManager.getNumberOfRequestsMade());
```

Comment:
So I finally figured out why I was uncomforable with this.  This method currently tracks ALL requests made.  That is, all requests since the server was started.  Is that really what you wanted?